package.json

```diff
@@ -65,7 +65,7 @@
         "@types/lodash.debounce": "^4.0.6",
         "@types/lodash.escape": "^4.0.6",
         "@types/node-fetch": "^2.5.8",
-        "@types/sanitize-html": "^1.18.3",
+        "@types/sanitize-html": "^1.27.1",
```

Comment:
Transitive dependency of sanitize-html caused a failure in generating the schema.